devops/lib/config.rb

```diff
@@ -0,0 +1,11 @@
+def application
+  'EDx'
+end
+
+def environment
+  'Stage'
+end
+
+def elasticsearch_domain
+  [application, environment].join('-').downcase
```

Comment:
This is a first pass at separating the configuration from the code, but this will eventually change drastically. Until there is a pattern that gets defined, I'm going to leave it for refactor later.